docs/ux/use_cases.md

```diff
@@ -62,3 +62,19 @@ When viewers support
 ["choice"](https://preview.iiif.io/cookbook/3333-choice/recipe/0033-choice/) such that we can interleave normal
 page scans with multispectral scans we hope to combine these resources for a
 better experience.
+
+## CollectorSystems
+
+PUL Preservation & Conservation uses
+[CollectorSystems](https://www.collectorsystems.com/index.htm) to manage their
+workflows. They've worked with a developer there to add custom import processes
+into CollectorSystems. APIs it requires are:
+
+1. Fetch all Figgy objects linked to a source metadata identifier which are
+   restricted
+   * https://figgy.princeton.edu/catalog?q=source_metadata_identifier_ssim:9934953393506421&format=json&f[visibility_ssim][]=restricted
```

Comment:
Do they use a token for this?

Reply:
Yes! I removed it from the docs, but mentioning that somehow would be good.

Reply:
Added auth token info.